lib/megaLogger.js

```diff
@@ -80,12 +78,26 @@
     };
 
     /**
+     * Static, converts enum to text/label
+     *
+     * @private
+     * @static
+     */
+    var _intToLevel = {
+        '40': 'ERROR',
+        '30': 'WARN',
+        '20': 'INFO',
+        '10': 'LOG',
+        '0': 'DEBUG'
+    };
+
+    /**
      * Static, global log registry
      *
      * @private
      * @static
      */
-    MegaLogger._logRegistry = {};
+    var _logRegistry = {};
```

Comment:
Since that is `@private`, it doesn't really need to be public. I've made this change in a couple of other places as you'll see below.